cgi-bin/ljtestlib.pl

```diff
@@ -0,0 +1,83 @@
+package LJ;
+
+use strict;
+no warnings 'uninitialized';
+
+
+BEGIN {
+    use Test::MockObject;
```

Comment:
You'll need to add any new dependencies to the `doc/dependencies-cpanm` file. That will auto-install them in the various environments. This will fix the first failure in the build.

Reply:
done

Reply:
done

Reply:
done